### ISSUE TITLE
Proposal to Rebrand Cardano Developer Portal to how.cardano.org

### ISSUE THREAD

**Issue by katomm** [cardano-foundation/developer-portal#1292]
The Cardano Developer Portal has grown to include a wealth of content that extends beyond developer-specific resources. It now encompasses various topics such as stake pool operations, governance, Cardano Improvement Proposals (CIPs), and showcases of Cardano projects. Despite this expansion, the name “Developer Portal” and the domain developers.cardano.org might be deterring non-developer members of the community from utilizing these resources. 

Proposal:
To make the portal more inclusive and representative of its broad scope, @gufmar and myself propose rebranding it to how.cardano.org. This new name better reflects the diverse range of content available and encourages participation from all community members, not just developers.

Rationale:

	1.	Inclusivity: A name like how.cardano.org signals that the portal is a comprehensive resource for anyone interested in learning how to engage with the Cardano ecosystem. 
	2.	Content Diversity: The portal already includes extensive information on governance, stake pool operations, CIPs, data analytics, and more, which can be highlighted better under a more inclusive name.
	3.	Community Engagement: Rebranding can attract a wider audience, fostering greater community participation and collaboration.
	4.	User Experience: An inclusive name can improve the user experience by clearly communicating that the portal is not here for everyone not only the developers. The platform works very well and it would be a lot of overhead to now start new platforms for similar things.

Implementation:

	•	Domain Transition: either implement redirects from developers.cardano.org to how.cardano.org to ensure a seamless user experience. Or add how.cardano.org and deep-link developers.cardano.org into the developer sections.
	•	Communication Plan: tbd, but basically announce the rebranding through community channels and update all references to the old domain in documentation and resources.
	•	Feedback and Iteration: Engage the Cardano community to gather feedback on the rebranding and make adjustments based on their input.

Feedback Requested:
Please share your thoughts on this rebranding proposal. Do you think it will make the portal more accessible and inclusive? Are there any potential challenges or considerations we should address? Would anyone be put off by this idea?

**Comment by rphair** [comment]
While thinking about this in work breaks there are only 2 things that stuck in my mind:
1. I agree with everything said in the OP about why the name needs to be changed to reflect a broader target audience... also realising that this has been subconsciously on my mind for a while.
2. Something initially bothered me about the name `how.cardano.org` but I couldn't put my finger on what... perhaps ambiguity compared to the precision of the things we usually work with.

Then I recalled the long running domain `why.cardano.org` and realised the `how` choice does make a lot of sense.  It suggests practicality across a wide range of possibilities, as well as a natural progression from `why` to `how` when considering any undertaking.  Any common alternatives like `info` would be equally ambiguous... and with a more specific title we'd likely end up with the original problem of only appealing to a narrow target audience.

So that's all my thoughts so far along with a confirmation that I would be in favour of the change exactly as it has been proposed.

**Comment by BlondeyGill** [comment]
As a Cardano "Enthusiast" who does frontend development (i.e. basic programming in comparison to blockchain dev) -- I have always wanted to learn about creating my own smart contracts, staking pools, or even building on the platform.

However, these have been my constant issues:
- The feeling of not enough time to devote to this
- This content is hardly ever in my face (i.e. YouTube or X.com)
- I never seen devs provide quick 1-3 minute videos teaching me this stuff

Here's what I like about this proposal:
- A rebrand is not only great for faster navigating of new users. This allows extra time and effort to be put forward about how to best display the information in an easy-to-digest way. There is an amazing repository at Cardano Foundation's developer portal but it's hard for me to follow as a newbie
- The effort put into this will be echoed on social media, as this proposal already has been which led me here. The more "echoes," the more users engaged in coming and building on Cardano
- I love the emphasis that is being put forward that this is for everyone, not just developers. People like me, who code but not "code," are hesitant to engage with these platforms because hell if I want to download a bunch of repositories and set up a local server to test code, etc.

One thing to consider is how the current portal is being used by current users. I.e. are developers currently engaging with the portal and will a rebrand affect their experience? No one who was using Windows XP liked the transition to Vista. Everything moved around and they couldn't find anything. If there is a negligible amount of users using the portal, this is null and void.

Another question to ask is... how will putting this content and rearranging it actually bring traffic? Yes, there will be the initial bump (like we see with meme coin purchasing), but then traffic dies down significantly. Some thought has to be put into either providing regular content that is shared on X.com (our strongest social media platform), or fun and inviting ways that bring people like me over to the site. (i.e. Charles doing a 5 minute video of building a stake pool, streaming the steps he's following on how.cardano.org) 

Either way, I think it's a great idea and support it fully.

**Comment by rdlrt** [comment]
I agree with the proposal name and intent - assuming it's locked down to cater for all prevent re-branding in future.

About usage - personally, I feel if someone wants to find something, they will find a way to (a simple domain name wouldnt stop them). Those who would complain about thing like domain name, will find next reason not to (for that section, name of the site will not matter).

Questions:
- Do we then ditch 'showcase' section? Does not seem to go well on 'how' , but I suppose not all the content 'has to' align to that name?
- Are there any proposals/intent about any further structural changes (eg: I hope we're not going back to idea of using point-in-time videos that become quickly outdated for how-to-run-a node for instance)?
- Since links to dev portal are included on other websites - the redirect will be essential, would there be a plan to retire developer.cardano.org domain at some point in future (I suppose it could be done based on traffic monitoring to the website)?

**Comment by rphair** [comment]
> Do we then ditch 'showcase' section? Does not seem to go well on 'how' , but I suppose not all the content 'has to' align to that name?

When I'm thinking of how to build something useful to users (i.e. from the developer perspective), I often make a survey of things others have built... to see "how" these look, "how" they feel, and "how" they appear to work.

And most of the Showcase items are _tools_ from the user perspective: i.e. means "how" the categorised things can be done.

So I think the Showcase qualifies as "how" regardless of the target audience.

**Comment by katomm** [comment]
> * Do we then ditch 'showcase' section? Does not seem to go well on 'how' , but I suppose not all the content 'has to' align to that name?

We would not ditch the showcase. Instead sending people to the "Developer Portal Showcase" I recommend to just send people to the "Cardano Showcase". I know the difference for someone who has been here for a long time seems marginal, but the effect on others is enormous.

> * Are there any proposals/intent about any further structural changes (eg: I hope we're not going back to idea of using point-in-time videos that become quickly outdated for how-to-run-a node for instance)?

No, I think the past has shown that this doesn't work.

> * Since links to dev portal are included on other websites - the redirect will be essential, would there be a plan to retire developer.cardano.org domain at some point in future (I suppose it could be done based on traffic monitoring to the website)?

Yes. That would not be a problem.

**Comment by katomm** [comment]
The old "developer showcase" was not moved (to be precise: copied) to https://cardano.org/apps and can be removed from the developer portal. There are a few open pull requests that need to be handled and we also need to change the documentation on this. https://github.com/cardano-foundation/cardano-org/issues/354
